### PR TITLE
ARM64-SVE: Fix SVE API formatting for FFR APIs

### PR DESCRIPTION
**Pull request by a74nh**
Missed from the recent APIs added

### CODE REVIEW

**Comment by a74nh** [comment]
@dotnet/arm64-contrib @kunalspathak

File: src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/Arm/Sve.PlatformNotSupported.cs [hunk 1]
```diff
@@ -4428,6 +4428,8 @@ public new abstract class Arm64 : AdvSimd.Arm64
         public static unsafe ulong GetActiveElementCount(Vector<ulong> mask, Vector<ulong> from) { throw new PlatformNotSupportedException(); }
 
 
+        // Read FFR, returning predicate of succesfully loaded elements
```
Comment (amanasifkhalid) [inline src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/Arm/Sve.PlatformNotSupported.cs:4431]:
```suggestion
        // Read FFR, returning predicate of successfully loaded elements
```

File: src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/Arm/Sve.cs [hunk 2]
```diff
@@ -4423,6 +4425,8 @@ public new abstract class Arm64 : AdvSimd.Arm64
         public static unsafe ulong GetActiveElementCount(Vector<ulong> mask, Vector<ulong> from) => GetActiveElementCount(mask, from);
 
 
+        // Read FFR, returning predicate of succesfully loaded elements
```
Comment (amanasifkhalid) [inline src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/Arm/Sve.cs:4428]:
```suggestion
        // Read FFR, returning predicate of successfully loaded elements
```

**Review by amanasifkhalid — COMMENTED** [review]
Thanks for fixing this! @dotnet/arm64-contrib it might be a good idea to follow this formatting in the last few API PRs.

File: src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/Arm/Sve.PlatformNotSupported.cs (hunk 1 above)
Comment (a74nh) [reply src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/Arm/Sve.PlatformNotSupported.cs:4431]: Oh dear, looks like this is wrong in the Arm Arm:
https://docsmirror.github.io/A64/2023-09/rdffr_p_p_f.html

**Review by amanasifkhalid — APPROVED** [review]

File: src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/Arm/Sve.PlatformNotSupported.cs (hunk 1 above)
Comment (a74nh) [reply src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/Arm/Sve.PlatformNotSupported.cs]: However, it has been fixed in the [latest version of the Arm Arm.](https://developer.arm.com/documentation/ddi0487/latest/)

**Pull request merged**